src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/EditorTopComponent.java

```diff
@@ -52,6 +51,10 @@
 import java.util.*;
 import javax.swing.*;
 import javax.swing.border.Border;
+import org.apache.batik.dom.GenericDOMImplementation;
+import org.apache.batik.svggen.SVGGeneratorContext;
+import org.apache.batik.svggen.SVGGraphics2D;
```

Comment:
When trying to build IGV with this patch I get
```
Compilation failure
[ERROR] /home/christian/jdk/open/src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/EditorTopComponent.java:[648,18] cannot find symbol
[ERROR]   symbol:   class SVGGraphics2DIOException
[ERROR]   location: class com.sun.hotspot.igv.view.EditorTopComponent
```
I think there is a missing `import org.apache.batik.svggen.SVGGraphics2DIOException;` here.